mipengine/node/node.py

```diff
@@ -1,23 +1,42 @@
+import argparse
+
 from celery import Celery
 
 from mipengine.common.node_catalog import node_catalog
-from mipengine.node.config.config_parser import config
+from mipengine import config
+
+if __name__ != "__main__":
```

Comment:
Refactor to:
```
if __name__ == "__main__":
    parser = argparse.ArgumentParser()
    parser.add_argument("--node-id", help="Current node identifier.", required=True)
    extra_args, celery_args = parser.parse_known_args()
    config.node.identifier = extra_args.node_id

node_id = config.node.identifier

```

This could work across the project. Please validate.